msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -0,0 +1,327 @@
+### YamlMime:Tutorial
+title: Teams file upload bot
+metadata:
+  title: Teams File Upload Bot 
+  description: In this tutorial, you'll learn how to upload files to Teams from a bot and how to receive a file sent to a bot as an attachment.
+  audience: Developer
+  level: Beginner
+  ms.date: 28/12/2021
+  ms.topic: interactive-tutorial
+  nextTutorialHref: bots/how-to/bots-filesv4.md
+  nextTutorialTitle: Read more on send and receive files through the bot
+  ms.custom: mvc
+  ms.localizationpriority: none
+items:
+- durationInMinutes: 1
+  content: |
+   The bot is created using Bot Framework. It shows how to upload files to Teams from a bot and how to receive a file sent to a bot as an attachment. 
+   It also shows how to fetch inline images sent in message.
+    
+   This step-by-step guide helps you to upload files from a bot to Microsoft Teams. You'll see the following output:
+
+   :::image type="content" source="assets/images/teams-file-upload-bot/output-sbs.png" alt-text="sbs output" lightbox="assets/images/teams-file-upload-bot/output-sbs.png":::
+   
+- title: Prerequisites
+  durationInMinutes: 1 
+  content: |
+    Ensure that you install the following tools and set up your development environment:  
+
+    * Microsoft Teams is installed and you have an account
+    * [.NET Core SDK version 3.1](https://dotnet.microsoft.com/en-us/download)
```

Comment:
Add URLs without en-us. Just check once that the links work without en-us language qualifier or not.

Reply:
@ashishguptaiitb,

Sure, we'll check.

Reply:
have made changes as advised